lib/packagekit-glib2/pk-progress.h

```diff
@@ -106,6 +107,7 @@ typedef enum {
 	PK_PROGRESS_TYPE_PACKAGE,
 	PK_PROGRESS_TYPE_ITEM_PROGRESS,
 	PK_PROGRESS_TYPE_TRANSACTION_FLAGS,
+	PK_PROGRESS_TYPE_SENDER,
```

Comment:
I think this will need to be inserted after `PK_PROGRESS_TYPE_INVALID` in the enum, or using a new PackageKit library with a client program which was compiled against an older version will result in any progress emissions of type `PK_PROGRESS_TYPE_SENDER` being interpreted by the client as `INVALID`.

Reply:
Hmm... I wonder whether that would cause problems if SENDER was considered invalid for older clients... But it's indeed better to be conservative here, good catch! I think while modifying it I didn't see this as public API, which was a bad call.